### PR TITLE
feat: add base e-mail template

### PR DESCRIPTION
**Pull request by xmod3905**
adding:
-base e-mail template

### CODE REVIEW

File: template/base.html [hunk 1]
```diff
@@ -1,12 +1,95 @@
...
+</head>
+<body style="background: #cce6e0" >
+    <table width="100%" border="0" cellspacing="0" cellpadding="0">
+        <tr align="center" style="font-family: Poppins, sans-serif; line-height: 1.25">
+            <td width="100%">
+                <div style="width: 100%; max-width: 705px; background: white; margin: 75px 0px; border-radius: 20px;">
+                    <table align="center" width="100%" style="font-weight:400">
+                        <tr style="color:#57585f; font-size:13px" align="center">
+                            <td>
+                                <img src="http://smandakebumen.sch.id/assets/img/tmp_mentoring/ganisda_logo.png" style="margin-top: 30px;" height="115px" alt="GANISDA Logo">
```
Comment (rizalarfiyan) [inline template/base.html]: tmp_mentoring?

File: template/base.html [hunk 2]
```diff
@@ -1,12 +1,95 @@
...
+                                </div>
+                            </td>
+                        </tr>
+                        <tr align="center">
+                            <td>
+                                <div style="width: 90%; margin: 25px; color: #637582">
+                                    <hr>
+                                    <h3>Ikuti Kami</h3>
+                                    <div class="social-container">
+                                        <a href="https://ganisdags.000webhostapp.com"><img src="http://smandakebumen.sch.id/assets/img/tmp_mentoring/social_logo/sm_button_linkedin.png" alt="Linkedin Button" height="35px"></a>
```
Comment (rizalarfiyan) [inline template/base.html]: URL udah bener?

File: template/base.html [hunk 3]
```diff
@@ -1,12 +1,95 @@
...
+                                    kali ini dengan sangat baik oleh karena itu kami memberikan penghargaan kepada kamu berupa 
+                                    <span style="color:#edb944;font-weight:600">E-Sertifikat Cerdas</span> 
+                                    yang juga terlampir pada surat elektronik ini
+                                </div>
+                                <div>Kamu juga bisa melihat <span style="color:#edb944;font-weight:600">E-Sertifikat Cerdas</span>-mu pada tombol dibawah ini dengan fitur yang lebih lengkap</div>
+                                <a href="(link)" style="background: #edb944;color: white;font-size:18px;padding: 6px 40px;border-radius:15px; text-decoration: none; margin: 15px 0px; display:inline-block">Yuk Lihat!</a>
+                                <div>atau kamu bisa pindai kode QR pada sertifikat!</div>
+                                <div>Pelajari lebih lanjut tantang E-Sertifikat Cerdas 
+                                    <a href="http://ganisdags.000webhostapp.com/e-sertifikat">disini!</a>.
```
Comment (rizalarfiyan) [inline template/base.html:66]: web masih coming soon

Comment (xmod3905) [reply template/base.html:66]: iya

**Review by rizalarfiyan — APPROVED** [review]

**Pull request merged**